code/modules/items/instruments/instruments.dm

```diff
@@ -227,6 +227,20 @@
 		else
 			src.play(user)
 
+	attack_ai(mob/user as mob)
+		..()
+		if (!in_interact_range(src, user)) // Instruments are not wireless
+			return
+
+		if (isAI(user))
+			var/mob/living/silicon/ai/borgo = user
+			if (borgo.deployed_to_eyecam)
```

Comment:
You should check if the user is an ai eye instead because user will not be the mainframe if they are